custom-vu/src/main/kotlin/jces1209/vu/page/boards/cloud/BoardPage.kt

```diff
@@ -0,0 +1,15 @@
+package jces1209.vu.page.boards.cloud
+
+import java.net.URI
+
+interface BoardPage {
+
+    val uri: URI
+    fun waitForBoardPageToLoad(): BoardContent
+    fun areThereIssues(): Boolean
+
+    /**
+     * Board must have issues
+     */
+    fun previewIssue(): BoardPage
```

Comment:
Since this can be done on both DC and Cloud - we should go back to the previous interface location: `BoardPage` and implement it for both products.

This way we'll have the same VU action, just using different page object implementations. Just like we did with `WorkAnIssue` and `AbstractIssuePage`:
https://github.com/atlassian/jces-1209/blob/3c414197e4fed170be0af9b91369f4ea53b1255a/custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt#L44-L45

- [ ] one VU action, one `BoardsPage` interface, two impls

Reply:
Thought about it and added to `technical debt` - https://hello.atlassian.net/wiki/spaces/JCJ/pages/730129567/Technical+debt
but if you insist let's implement this one in this review

Reply:
1. it's fastest to fix it now, while we have a fresh mental context
2. tech debt backlogs tend to not be done - Confluence pages even faster
3. `master` doesn't have this debt, so now is the best time to prevent it